{{cookiecutter.python_name}}/pyproject.toml

```diff
@@ -79,7 +79,7 @@ version_cmd = "hatch version"
 
 [tool.jupyter-releaser.hooks]
 before-build-npm = [
-    "python -m pip install jupyterlab>=4.0.0a29,<5",
+    "python -m pip install jupyterlab~=4.0.0a29",
```

Comment:
```suggestion
    "python -m pip install 'jupyterlab>=4.0.0a29,<5'",
```
should do it

Reply:
I have executed `pip install jupyterlab~=4.0.0a29` and `python -m pip install 'jupyterlab>=4.0.0a29,<5'`
Both of them work well.